lightning/src/chain/keysinterface.rs

```diff
@@ -51,29 +52,30 @@ use crate::util::invoice::construct_invoice_preimage;
 
 /// Used as initial key material, to be expanded into multiple secret keys (but not to be used
 /// directly). This is used within LDK to encrypt/decrypt inbound payment data.
-/// (C-not exported) as we just use [u8; 32] directly
+///
+/// (C-not exported) as we just use `[u8; 32]` directly
 #[derive(Hash, Copy, Clone, PartialEq, Eq, Debug)]
 pub struct KeyMaterial(pub [u8; 32]);
 
-/// Information about a spendable output to a P2WSH script. See
-/// SpendableOutputDescriptor::DelayedPaymentOutput for more details on how to spend this.
+/// Information about a spendable output to a `P2WSH` script.
+///
+/// See [`SpendableOutputDescriptor::DelayedPaymentOutput`] for more details on how to spend this.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct DelayedPaymentOutputDescriptor {
-	/// The outpoint which is spendable
+	/// The outpoint which is spendable.
```

Comment:
Mh, while it stands out in this case because the line is so short, I still think it should have a period, in particular to maintain consistency with other examples with longer lines, in which we def. want a period. E.g.:

```
	/// An output to a P2WPKH, spendable exclusively by our payment key (i.e., the private key
	/// which corresponds to the `payment_point` in [`BaseSign::pubkeys`]). The witness
	/// in the spending input is, thus, simply:
```